projects/bootloader/src/jump_to_application_dispatcher.c

```diff
@@ -0,0 +1,49 @@
+#include "jump_to_application_dispatcher.h"
+
+#include <stdint.h>
+
+#include "bootloader_crc32.h"
+#include "jump_to_application.h"
+
+#include "bootloader_can.h"
+#include "bootloader_datagram_defs.h"
+#include "can_datagram.h"
+#include "config.h"
+#include "dispatcher.h"
+#include "log.h"
+
+// response config datagram setup
+static uint8_t s_status;
+static CanDatagramTxConfig s_response_config = {
+  .dgram_type = BOOTLOADER_DATAGRAM_JUMP_TO_APP,
```

Comment:
This should just be `BOOTLOADER_DATAGRAM_STATUS_RESPONSE`

Reply:
done